public/sass/pages/_explore.scss

```diff
@@ -273,6 +277,11 @@
   flex-grow: 1;
 }
 
+.query-row-break {
```

Comment:
If we can we should probably not add additional classes and rather use `emotion`. May be a bit awkward to combine normal css and emotion css but probably only way to move the migration forward.